### DISCUSSION TITLE
How to create a standalone game?

### DISCUSSION THREAD

**Discussion by Sectonid** [nillerusr/source-engine#369] [Q&A] [answered]
So, what i know about this fork is that it's supposed to be compatible with most of Valve's Source Engine Games. However, because this is a leaked engine, its code is fully available and therefore allows to create a standalone game, rather than a mod. Is there any way to create a standalone game?
Edit: to be more clear, i want to create a game that compiles without any external Source Engine game.

**Comment by mittorn** [comment, marked as answer]
Making standalone game with source sdk not running on Source Engine is illegal. Making standalone game with souce sdk running on leaked Source Engine is double illegal!

**Reply by Sectonid** [reply]
Oh, ok then.
